airbyte-integrations/connectors/destination-bigquery/src/test-integration/kotlin/io/airbyte/integrations/destination/bigquery/BigqueryWriteTest.kt

```diff
@@ -74,8 +73,8 @@ abstract class BigqueryTDWriteTest(configContents: String) :
         configContents = configContents,
         BigqueryFinalTableDataDumper,
         ColumnNameModifyingMapper(BigqueryColumnNameGenerator())
+            .compose(TimeWithTimezoneMapper)
             .compose(RootLevelTimestampsToUtcMapper)
-            .compose(TypingDedupingMetaChangeMapper)
```

Comment:
don't need this anymore - now that we're doing typing in-connector, we get the correct Meta.Change.Reason values.